bindings_ffi/src/mls.rs

```diff
@@ -166,7 +171,16 @@ pub async fn get_inbox_id_for_address(
     api: Arc<XmtpApiClient>,
     account_address: String,
 ) -> Result<Option<String>, GenericError> {
-    let api = ApiClientWrapper::new(Arc::new(api.0.clone()), Default::default());
+    let cooldown = xmtp_common::ExponentialBackoff::builder()
+        .duration(std::time::Duration::from_secs(3))
```

Comment:
this is where the rate limit cooloff is configured. Wasn't completely sure about the values here, but 3 seconds that multiplies by three for every consecutive rate limit sounds reasonable for now.

Reply:
nit: I suggest to extract the value as a value somewhere else, like in the configuration file? Also since the rate limit on the BE configured for 5 minutes, that's still a huge gap, what we should do about it? prob a question for @neekolas

Reply:
Extracting to a place in xmtp_api is reasonable, will update for that